### PR TITLE
Rename old FIXME to TODO

### PR DESCRIPTION
**Pull request by elit0451**
## Details
Rename old _FIXME_ to _TODO_, so we can track new _FIXME_ starting with v13.

### CODE REVIEW

**Review by nikolajlauridsen — COMMENTED** [review]
LGTM, but one of the TODOS are empty, so maybe we should just remove it?

File: tests/Umbraco.Tests.Integration/Umbraco.Infrastructure/Persistence/NPocoTests/NPocoFetchTests.cs [hunk 1]
```diff
@@ -310,7 +310,7 @@ public void TestCalculated()
             //    GROUP BY zbThing1.id, zbThing1.name";
             var sql = ScopeAccessor.AmbientScope.SqlContext.Sql()
                 .Select<Thing1Dto>()
-                .Append(", COUNT(zbThing2Group.groupId) AS groupCount") // FIXME:
```
[the remark this answers is not in the document]
Comment (Zeegaan) [reply tests/Umbraco.Tests.Integration/Umbraco.Infrastructure/Persistence/NPocoTests/NPocoFetchTests.cs:313]: Agreed 👍

[the remark this answers is not in the document]
Comment (elit0451) [reply tests/Umbraco.Tests.Integration/Umbraco.Infrastructure/Persistence/NPocoTests/NPocoFetchTests.cs:313]: Fixed

**Review by nikolajlauridsen — APPROVED** [review]
Looks good to me now, will merge when lights go green 😄

**Pull request merged**